nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/magick/utils/ImageResizer.java

```diff
@@ -90,7 +90,7 @@ public static Point scaleToMax(int width, int height, int max) {
             float ratio = maxSide / max;
             return new Point(Math.round(width / ratio), Math.round(height / ratio));
         }
-        return null;
+        return new Point(width, height);
```

Comment:
Worth adding a Javadoc comment to state that it now returns the original `width` and `height` if none is `> max`.

Reply:
Done!